fatal error: 'React/RCTBridgeModule.h' file not found

When attempting to archive the app on iOS, I get 

`RNBackgroundTimer.h:9:9: fatal error: 'React/RCTBridgeModule.h' file not found`

Environment:
  OS: macOS High Sierra 10.13.3
  Node: 8.9.4
  Yarn: 1.3.2
  npm: 5.6.0
  Watchman: 4.9.0
  Xcode: Xcode 9.3.1 Build version 9E501
  Android Studio: 3.0 AI-171.4443003

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: 0.55.4 => 0.55.4

--- a/react-native-background-timer.podspec
+++ b/react-native-background-timer.podspec
@@ -19,4 +19,6 @@ Pod::Spec.new do |s|
   s.preserve_paths = 'README.md', 'package.json', 'index.js'
   s.source_files   = 'ios/*.{h,m}'
 
+  s.dependency 'React'
+
 end

